crates/api_models/src/payouts.rs

```diff
@@ -52,15 +50,15 @@ pub struct PayoutCreateRequest {
     }))]
     pub routing: Option<serde_json::Value>,
 
-    /// This allows the merchant to manually select a connector with which the payout can go through
+    /// This field allows the merchant to manually select a connector with which the payout can go through.
     #[schema(value_type = Option<Vec<PayoutConnectors>>, max_length = 255, example = json!(["wise", "adyen"]))]
     pub connector: Option<Vec<api_enums::PayoutConnectors>>,
 
-    /// The boolean value to create payout with connector
+    /// This field is used when merchant wants to confirm the Payout, thus useful for the Payout Confirm Request. Ideally Merchants should create a Payout, Update it(if required), then Confirm it.
     #[schema(value_type = bool, example = true, default = false)]
     pub confirm: Option<bool>,
 
-    /// The payout_type of the payout request can be specified here
+    /// The payout_type of the payout request can be specified here, this is a mandatory field to Confirm the Payouts, i.e., Should be passed in Create request, if not then should be updated in the Payout Update request, then only it can be confirmed.
```

Comment:
```suggestion
    /// The payout_type of the payout request can be specified here, this is a mandatory field to _Confirm_ the payout, i.e., should be passed in _Create_ request, if not then should be updated in the payout _Update_ request, then only it can be confirmed.
```